modules/sdk-coin-apt/src/apt.ts

```diff
@@ -46,28 +50,54 @@ export class Apt extends BaseCoin {
     return 'Aptos';
   }
 
-  verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
-    throw new Error('Method not implemented.');
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return true;
+  }
+
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'eddsa';
   }
 
-  isWalletAddress(params: VerifyAddressOptions): Promise<boolean> {
+  allowsAccountConsolidations(): boolean {
+    return true;
+  }
+
+  async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
     throw new Error('Method not implemented.');
   }
 
+  async isWalletAddress(params: VerifyAddressOptions): Promise<boolean> {
+    const { address: newAddress } = params;
+
+    if (!this.isValidAddress(newAddress)) {
+      throw new InvalidAddressError(`invalid address: ${newAddress}`);
+    }
+    return true;
+  }
```

Comment:
isn't this method supposed to ensure that the address is an actual _wallet_ address as opposed to an arbitrary address?